wallet/src/main/scala/org/bitcoins/wallet/internal/UtxoHandling.scala

```diff
@@ -300,35 +316,19 @@ private[wallet] trait UtxoHandling extends WalletLogger {
     require(unreserved.isEmpty, s"Some utxos are not reserved, got $unreserved")
 
     // unmark all utxos are reserved
-    val groupedUtxos = utxos
+    val updatedUtxos = utxos
       .map(_.copyWithState(TxoState.PendingConfirmationsReceived))
```

Comment:
Why is the `TxoState.PendingConfirmationsReceived` received here? 

Is this the "safest" state to use in this situation?

Reply:
Pretty much